utils/spacewalk-common-channels.ini

```diff
@@ -1534,6 +1590,16 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-x86_64-Media1/
 
+[uyuni-server-stable-leap-153]
```

Comment:
I wonder if we should keep this commented for now, and same for the proxy.

Otherwise people could think that this already works, which is not the case.

Reply:
I don't know ... whatever you think is best, I don't have a strong opinion on it